Example/ViewController+FreeDrive.swift

```diff
@@ -70,6 +70,7 @@ extension ViewController {
         
         navigationMapView.mapView.on(.styleLoaded, handler: { [weak self] _ in
             guard let self = self else { return }
+            self.navigationMapView.localizeLabels()
```

Comment:
We were also calling `localizeLabels()` in `RouteMapViewController`, `CarPlayMapViewController` and `CarPlayNavigationViewController` in `1.x`.

Reply:
It is still being called in RouteMapViewController, CarPlayMapViewController, and CarPlayNavigationViewController, but it must’ve fallen out of the example application by accident.